doc/source/hooks.rst

```diff
@@ -1,7 +1,7 @@
 Hooks
 =====
 
-From version ``0.12`` Papis has a minimal hook infrastructure.
+From version ``v0.12`` Papis has a minimal hook infrastructure.
```

Comment:
Any opinion how we should denote version numbers?

Made them consistent with the `v` at the start, and all wrapped in double backticks. Some didn't have `v`, some were italicized instead.

Reply:
P.S. just accept my change if it's fine like this :smile: I don't wanna be a bikeshed nuisance.

Reply:
Can't say I have much of an opinion either way, so this is fine!